grouper/fe/templates/service.html

```diff
@@ -0,0 +1,318 @@
+{% extends "base.html" %}
+{% from 'macros/ui.html' import group_panel, member_panel, permission_panel, log_entry_panel,
+                                help_for, account, public_key_panel, shell_panel, tokens_panel, %}
+
+{% block heading %}
+    <a href="/svc">Service Accounts</a>
+{% endblock %}
+
+{% block subheading %}
+    {{ user.username }}
+    {% if user.role_user %}<small>(role)</small>{% endif %}
+    {% if not user.enabled %}<small>(disabled)</small>{% endif %}
+
+    {% if group.audit and not group.audit.complete %}
```

Comment:
I think an SA audit would be meaningful, because anyone in the SA group either A) inherits the group's permissions, or B) can pretend to be the SA, and therefore have that group's permissions.